packages/neon/neon_dashboard/lib/src/pages/main.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:neon/blocs.dart';
+import 'package:neon/utils.dart';
+import 'package:neon/widgets.dart';
+import 'package:neon_dashboard/src/blocs/dashboard.dart';
+import 'package:neon_dashboard/src/widgets/widget.dart';
+
+class DashboardMainPage extends StatelessWidget {
+  const DashboardMainPage({
+    super.key,
+  });
+
+  @override
+  Widget build(final BuildContext context) {
+    final bloc = NeonProvider.of<DashboardBloc>(context);
+
+    return ResultBuilder.behaviorSubject(
+      stream: bloc.widgets,
+      builder: (final context, final snapshot) {
+        Widget? child;
+        if (snapshot.hasData) {
+          child = Wrap(
+            alignment: WrapAlignment.center,
+            spacing: 8,
+            runSpacing: 8,
+            children: snapshot.requireData.entries
+                .map(
+                  (final widget) => DashboardWidget(
+                    widget: widget.key,
+                    items: widget.value,
+                  ),
+                )
+                .toList(),
+          );
+        }
+
+        return Center(
+          child: NeonListView.custom(
+            shrinkWrap: true,
```

Comment:
I still don't understand why we need shrinkwrap.
The children (`DashboardWidget` in this case) have a fixed size.

Or are there use cases that I just don't see?

Reply:
The whole list should be centered on the page. You can't center the items of a list, so shrinkWrap and Center is needed to center the whole list.

Reply:
In this case
```dart
        return NeonListView.custom(
          isLoading: snapshot.isLoading,
          error: snapshot.error,
          onRefresh: bloc.refresh,
          sliver: SliverFillRemaining(
            hasScrollBody: false,
            child: Center(
              child: child,
            ),
          ),
        );
```

Should also do the trick.

Reply:
Oh I see, I will try it out. I should definitely get more into slivers.

Reply:
This works like a charm! Super nice :heart: